src/main/java/pl/confitura/jelatyna/admin/Admin.java

```diff
@@ -0,0 +1,73 @@
+package pl.confitura.jelatyna.admin;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import org.hibernate.validator.constraints.Email;
+import org.hibernate.validator.constraints.NotEmpty;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+@Data
+@EqualsAndHashCode
+@ToString
```

Comment:
@Data adds EqualsAndHashCode and ToString automatically

from docs:
@Data is a convenient shortcut annotation that bundles the features of @ToString, @EqualsAndHashCode, @Getter / @Setter and @RequiredArgsConstructor together